presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -51,6 +77,34 @@ public static double abs(@SqlType(StandardTypes.DOUBLE) double num)
         return Math.abs(num);
     }
 
+    private static SqlScalarFunction decimalAbsFunction()
```

Comment:
when I don't need to a pass extra parameters, that's certainly a better choice, thanks, in this case I can live without them. Done.